File: backend-csharp/Models/Pessoa.cs
namespace PagamentosApp.Models
{
    public class Pessoa
    {
        public int Id { get; set; }
        public string Nome { get; set; } = string.Empty;
        public int Idade { get; set; }
        public string Ramo { get; set; } = string.Empty; // Lobinho ou Escoteiro
        public string? FotoUrl { get; set; } // opcional
    }
}

File: frontend-flutter/lib/models/pessoa.dart
class Pessoa {
  int id;
  String nome;
  int idade;
  String ramo;
  String? fotoUrl;

  Pessoa({
    required this.id,
    required this.nome,
    required this.idade,
    required this.ramo,
    this.fotoUrl,
  });

  factory Pessoa.fromJson(Map<String, dynamic> json) => Pessoa(
        id: json['id'],
        nome: json['nome'],
        idade: json['idade'],
        ramo: json['ramo'],
        fotoUrl: json['fotoUrl'],
      );

  // 🔽 Getter que monta a URL completa
  String? get fotoCompleta {
    if (fotoUrl == null || fotoUrl!.isEmpty) return null;
    return 'http://192.168.2.134:5000/$fotoUrl';
  }
}
